### ISSUE TITLE
Package.appxmanifest cannot be opened in Visual Studio 2022

### ISSUE THREAD

**Issue by mobynet1** [unoplatform/uno.templates#876]
### Current behavior

I am trying to follow the instructions on how to manually add a splash screen to my application. Here is yet another bug in the basic, minimal c# solution generated by Visual Studio 2022. I click on the Package.appxmanifest file in the solution explorer and get this error:

![Screenshot 2024-07-28 064110](https://github.com/user-attachments/assets/760039a1-0be6-4d48-a19f-06611c5830dc)

Do you guys even test this stuff before it gets released? It does not appear so.

### Expected behavior

I am trying to learn Uno, but the bugs in your setup are making it very difficult. I am beginning to lose faith that Uno is the direction I should go in my attempt to find a replacement for .NET Maui. When I create a brand new solution either with your online template wizard or via Visual Studio 2022, it should just work. I have not modified anything so I have no one to blame other than your QA department...

### Workaround

If out-ot-the-box stuff does not work, I am not thrilled with the prospect of trying to find work arounds.

### Works on UWP/WinUI

None

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### Affected platforms

Windows (WinAppSDK)

### IDE version

VS Code 1.91.1; Visual Studio 2022 Version 17.10.4

### Anything else we need to know?

Please test your code before it is released...

**Comment by MartinZikmund** [comment]
On my side I had to add:

```
<mp:PhoneIdentity PhoneProductId="SOMEGUID" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
```

To make package manifest to open up, apparently `PhoneIdentity` is still required by VS (Whyyyy 😂 )

**Comment by mobynet1** [comment]
@jeromelaban I apologize for getting snarky. I mentioned that I have been under severe pressure to get our alpha demo ready and the Maui team's inability to provide a workable product has left me simply exhausted. After fighting Maui issues all last week, I decided to explore Uno and finding issues here was a bit disheartening. I feel the response I have personally received from you has been excellent and I appreciate your attention. I promise to dial it back! :-)

**Comment by dansiegel** [comment]
I'll be making a PR shortly. Note that at build time, the Package.appxmanifest is updated to include all of the image assets. Additional fields within the Identity, Properties, etc are all provided via MSBuild properties that you see in the Single Project.

```xml
<PropertyGroup>
    !-- Display name -->
    <ApplicationTitle>MyExtensionsApp.1</ApplicationTitle>
    <!-- App Identifier -->
    <ApplicationId>com.companyname.myextensionsapp</ApplicationId>
    <!-- Versions -->
    <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
    <ApplicationVersion>1</ApplicationVersion>
    <!-- Package Publisher -->
    <ApplicationPublisher>Uno Platform</ApplicationPublisher>
    <!-- Package Description -->
    <Description>MyExtensionsApp.1 powered by Uno Platform.</Description>
</PropertyGroup>
```

In the mean time if you update your Package.appxmanifest as follows Visual Studio should be happy and it will still be properly updated at build time by Uno.Resizetizer.

```xml
<?xml version="1.0" encoding="utf-8"?>
<Package
  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
  IgnorableNamespaces="uap rescap">

  <!-- Provided by MSBuild Properties via Uno.Resizetizer -->
  <Identity />
  <Properties />

  <Dependencies>
    <!-- Proper values updated and set by Uno.Resizetizer -->
    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
  </Dependencies>

  <Resources>
    <Resource Language="x-generate"/>
  </Resources>

  <Applications>
    <Application Id="App"
      Executable="$targetnametoken$.exe"
      EntryPoint="$targetentrypoint$">
      <!-- Values provided by Uno.Resizetizer -->
      <uap:VisualElements />
    </Application>
  </Applications>

  <Capabilities>
    <rescap:Capability Name="runFullTrust" />
  </Capabilities>
</Package>
```

**Comment by dansiegel** [comment]
@mobynet1 this should be available in the templates from the CLI with 5.4.0-dev.151 or later which should be available soon. The fix is being backported and should be in the next service Release for 5.3.

**Comment by mobynet1** [comment]
@dansiegel Great job, Dan!! Very impressive turnaround!
